content/chainguard/chainguard-images/getting-started/laravel.md

```diff
@@ -0,0 +1,324 @@
+
+First, look for the container running the `app` service and copy its ID.
+
+```shell
+docker ps
+```
+```shell
+CONTAINER ID   IMAGE                                   COMMAND                  CREATED          STATUS          PORTS                                       NAMES
+4d8534e540cd   cgr.dev/chainguard/mariadb              "/usr/local/bin/dock…"   25 seconds ago   Up 24 seconds   0.0.0.0:3306->3306/tcp, :::3306->3306/tcp   octo-facts-mariadb-1
+cb24b8b3d96a   cgr.dev/chainguard/laravel:latest-dev   "/bin/s6-svscan /sv"     25 seconds ago   Up 24 seconds                                               octo-facts-app-1
+f6cc3978e0e8   cgr.dev/chainguard/nginx                "/usr/sbin/nginx -c …"   25 seconds ago   Up 24 seconds   0.0.0.0:8000->8080/tcp, :::8000->8080/tcp   octo-facts-nginx-1
+
+```
+Then, you can run migrations like this:
+
+```shell
+docker exec cb24b8b3d96a php /app/artisan migrate --seed
+```
+
+You can use the same method to execute other Artisan commands while the environment is up. After running migrations and seeding the database, you should be able to reload the app from your browser at `localhost:8000` and get a new Octopus fact.
+
+## 3. Creating a Distroless Laravel Runtime for the Application
+So far, we have been using the `laravel:latest-dev` builder image to run the application in a development setting. For production workloads, the recommended approach for additional security is to create a [distroless](/chainguard/chainguard-images/getting-started-distroless/) runtime for the application that will contain only what's absolutely necessary for running the app on production. This is done by combining a **build** phase in a **multi-stage** Dockerfile.
+
+To demonstrate this approach, we'll now build a distroless image and test it using the Docker Compose setup exemplified in the previous section.
+
+The following Dockerfile is included within the root of the application:
+
+```Dockerfile
+FROM cgr.dev/chainguard/laravel:latest-dev AS builder
+USER root
+RUN apk update && apk add nodejs npm
+COPY . /app
+RUN cd /app && chown -R php.php /app
+USER php
+RUN composer install --no-progress --no-dev --prefer-dist
+RUN npm install && npm run build
+
+FROM cgr.dev/chainguard/laravel:latest
+COPY --from=builder /app /app
+```
+
+This Dockerfile starts with a build stage that copies the application files to the container, installs Node and NPM, installs the application dependencies, and dumps front-end assets to the public folder. A second stage based on `laravel:latest` copies the application from the build stage to the final distroless image. It's important to notice that we don't run any database migrations here, because at build time the database might not be ready yet. In other production scenarios, you may be able to include the migration within the Dockerfile, as long as the database is already up and running.
+
+To build this image, run:
+
+```shell
+docker build . -t octofacts
```

Comment:
I think it would be better to replace the image tag in the compose file with `build: .` which automatically build the image when it's out-of-date.